.travis.yml

```diff
@@ -1,39 +1,44 @@
 language: python
 sudo: True
 cache: pip
-python:
-    - '3.6'
 os:
     - linux
 dist: xenial
+python:
+  - 2.7
```

Comment:
this is django 1.11 only so includ in django 1.11 was enough

Reply:
what do you mean?

Reply:
Ask for review before merging from now on.

Reply:
Sure, I can do that, but what do you mean by "this is django 1.11 only so includ in django 1.11 was enough" ?